webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -143,6 +143,7 @@
 
     {% for status in status_messages %}
     <div class="status-message severity-{{ status.severity }}">
+        <span>(<time datetime="{{ status.date.isoformat() }}" class="ago">{{ status.date.strftime('%a, %b %d %H:%M %Z') }}</time>)</span>
```

Comment:
Yeah, that sounds good. There's a few places where we define ``<time>`` tags, and they are not consistent at all already. I'll make them all use the new helper.